app/src/components/submissionPage/SnapSubmissionStyles.ts

```diff
@@ -0,0 +1,100 @@
+import styled from "styled-components";
+import { MentionsInput } from "react-mentions";
+
+/* A list of styled components for SubmissionTextBox.tsx */
+
+export const ElleImg = styled.img`
+    width: 100%;
```

Comment:
This actually gets overwritten by the `w-100` class applied to `ElleImg` when it is used. It might be better to move these styles directly to where they are used.